README.md

```diff
@@ -84,14 +84,73 @@ module.exports = {
 It is also possible to disable specific rules (or all rules) in a template itself:
 
 ```hbs
-{{! disable all rules for this template }}
-<!-- template-lint disable=true -->
+<!-- disable all rules -->
+{{! template-lint-disable }}
 
-{{! disable specific rules for this template }}
-<!-- template-lint bare-strings=false -->
+<!-- disable only bare-strings -->
+{{! template-lint-disable bare-strings }}
+
+<!-- disable only bare-strings and triple-curlies -->
+{{! template-lint-disable bare-strings triple-curlies }}
+
+<!-- enable all rules -->
+{{! template-lint-enable }}
+
+<!-- enable only bare-strings -->
```

Comment:
Does this enable only `bare-strings` or just enable bare-strings in addition to whatever else was enabled (I believe it does the latter?)?